src/test/java/dk/kb/present/transform/XSLTPreservicaToSolrTransformerTest.java

```diff
@@ -0,0 +1,93 @@
+package dk.kb.present.transform;
+
+import dk.kb.present.TestUtil;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+
+/**
+ *
+ *  To run these tests, the test metadata has to be fetched from the internal aegis project.
+ *  With aegis running this can be done by running 'kb init' in this repository.
+ */
+public class XSLTPreservicaToSolrTransformerTest {
+
+    public static final String PRESERVICA2SOLR = "xslt/preservica2solr.xsl";
+    public static final String RECORD_44979f67 = "internal_test_files/tvMetadata/44979f67-b563-462e-9bf1-c970167a5c5f.xml";
+    public static final String RECORD_5a5357be = "internal_test_files/tvMetadata/5a5357be-5890-472a-a294-41a99f108936.xml";
+
+    @Test
+    public void testExtraction() throws Exception {
+        String solrString = TestUtil.getTransformedWithAccessFieldsAdded(PRESERVICA2SOLR, RECORD_44979f67);
```

Comment:
These unit tests should not fail if the test files are not available but instead log a warning that they cannot be evaluated. I have a "clever idea" for this, so I'll make a pull request for it later.